Assets/IntegrationTests/TestShopifyCollections.cs

```diff
@@ -36,5 +37,41 @@ public class TestShopifyCollections : MonoBehaviour {
 
             Assert.IsTrue (waiter.IsStopped, Utils.MaxQueryMessage);
         }
+
+        [UnityTest]
+        public IEnumerator LoadAllCollections() {
+            float maxDuration = 6f;
+            int maxCollections = 13;
+            StoppableWaitForTime waiter = new StoppableWaitForTime (maxDuration);
+            ShopifyError errorsFromQueries = null;
+            List<Collection> AllCollections = new List<Collection>();
```

Comment:
We should be using type inference here where possible

Reply:
Hmmm... Why's that?

Reply:
Type inference makes the code more concise, especially where it is clear from the declaration what type the resulting variable will be. I was looking at https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions in their implicit local var section

Reply:
Yeah. In this case it might be cool to do. I just hate using `var` on a function return because I think it makes the code less readable especially considering that this is a public repo.